projectDocs/dev/codingStandards.md

```diff
@@ -19,45 +25,48 @@ Python code style is enforced with the Ruff linter, see [linting](../testing/aut
 Files can be checked out locally using CRLF if needed for Windows development using [git](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf).
 
 ### Indentation
+
 * Indentation must be done with tabs (one per level), not spaces.
 * When splitting a single statement over multiple lines, just indent one or more additional levels.
   Don't use vertical alignment; i.e. lining up with the bracket on the previous line.
-  - Be aware that this requires a new-line after an opening parenthesis/bracket/brace if you intend
+  * Be aware that this requires a new-line after an opening parenthesis/bracket/brace if you intend
     to split the statement over multiple lines.
```

Comment:
Ironically I think we try to make markdown sentences full lines rather than split them

```suggestion
  * Be aware that this requires a new-line after an opening parenthesis/bracket/brace if you intend to split the statement over multiple lines.
```